infrastructure/src/main/java/org/corfudb/infrastructure/IServerRouter.java

```diff
@@ -24,4 +26,9 @@
      * @param server    The server to route messages to
      */
     void addServer(AbstractServer server);
+
+    /**
+     * Get a list of registered servers.
+     */
+    List<AbstractServer> getServers();
```

Comment:
ImmutableList

Reply:
Why did you decide to expose a list of servers?

Reply:
Even `addServer` method is deprecated

Reply:
Also the current code already exposed getServers() logic in NettyServerRouter (lombok getter), I just pushed the logic to interface level.

Reply:
As discussed, keep the current behavior, router shouldn't be aware of sealing.